kernel/acpi/acpi_table/src/lib.rs

```diff
@@ -39,8 +36,8 @@ pub struct TableLocation {
 /// All ACPI tables are covered by a single large MappedPages object, 
 /// which is necessary because they may span multiple pages/frames,
 /// and generally should not be multiply aliased/accessed due to potential race conditions.
-/// As more ACPI tables are discovered, the single MappedPages object is 
-pub struct AcpiTables {
+/// As more ACPI tables are discovered, the single MappedPages object is
+pub struct AcpiTables { //^FIXME: is what?
```

Comment:
```suggestion
///
/// As more ACPI tables are discovered, the single MappedPages object is
/// extended to cover them.
pub struct AcpiTables {
```

Reply:
oops! lol, thanks for catching that. What @tsoutsman wrote is sufficient.